### PR TITLE
Reposition search

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Reposition search and make header taller

## QA

- Run ./run to get started
- Go to http://localhost:8015/core
- Make sure search is right aligned on large and that small works as expected
- The styling is as per live but I’m going to take a second pass at this to make it better

### CODE REVIEW

File: templates/includes/base/header.html [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
+            <a href="#main-content">Jump to main content</a>
+        </span>
+    </nav>
+
+    <form id="search" action="/search" class="header-search">
+        <label for="edit-keys" class="header-search__label">Search</label>
+        <input class="header-search__field" type="search" name="q" id="search" placeholder="Search the site" />
+        <button type="submit" class="header-search__submit">
+            <img src="{{ ASSET_SERVER_URL }}2196e362-search-white.svg" class="header-search__image--large-screen" alt="Perform a search" />
+            <img src="{{ ASSET_SERVER_URL }}19750a6d-search-black.svg" class="header-search__image--small-screen" alt="Perform a search" />
```
Comment (anthonydillon) [inline templates/includes/base/header.html]: Could you make this an optimised inline SVG and change the fill colour based on a media query? To save a request.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
One suggestion in code

**Comment by anthonydillon** [comment]
I know you said you were going for a second pass but I will mention that the placeholder colour should be white:
welcome - developer

On medium screens, the search hover area does not take up the entire height of the navigation:
welcome - developer 1

Also, the small screen navigation does not work.

**Comment by anthonydillon** [comment]
Thanks, @grahambancroft, looking good. Few comments above.

File: templates/includes/base/header.html (hunk 1 above)
Comment (grahambancroft) [reply templates/includes/base/header.html]: These were already in place, I think they’re just white space changes, in the interest of MVP would it be okay for me to add the improvement as an issue?

**Comment by grahambancroft** [comment]
Travis CI fix here https://github.com/vanilla-framework/vanilla-framework/pull/1050

**Comment by grahambancroft** [comment]
Added placeholder and focus color to input

**Comment by grahambancroft** [comment]
Fixed search hover, this was an issue on live also

File: templates/includes/base/header.html (hunk 1 above)
Comment (anthonydillon) [reply templates/includes/base/header.html]: Sure

File: templates/includes/base/header.html [hunk 2]
```diff
@@ -1,32 +1,29 @@
...
-            <label for="edit-keys" class="header-search__label">Search</label>
-            <input class="header-search__field" type="search" name="q" id="search" placeholder="Search the site" />
-            <button type="submit" class="header-search__submit">
-                <img src="{{ ASSET_SERVER_URL }}2196e362-search-white.svg" class="header-search__image--large-screen" alt="Perform a search" />
-                <img src="{{ ASSET_SERVER_URL }}19750a6d-search-black.svg" class="header-search__image--small-screen" alt="Perform a search" />
-            </button>
-        </form>
-    </div>
+    <form id="search" action="/search" class="header-search">
+        <label for="edit-keys" class="header-search__label">Search</label>
```
Comment (anthonydillon) [inline templates/includes/base/header.html]: For="search"?

Comment (grahambancroft) [reply templates/includes/base/header.html]: Fixed. Not entirely sure how this search stuff got through QA in the first place tbh

**Comment by anthonydillon** [comment]
Looking good but one issue. On 820px the search box is toggled but the text is still white.
core - developer

**Comment by grahambancroft** [comment]
Fixed

**Comment by anthonydillon** [comment]
I don't see a commit with the fix. Do you still have to push?

**Review by anthonydillon — APPROVED** [review]
Code and QA :+1:

**Pull request merged**